Extensions/probing.hh

```diff
@@ -136,7 +136,7 @@ inline double Gammadist(double data, double shape, double loc, double scale) {
          exp(-(1 / scale) * (data - loc)) / tgamma(shape);
 }
 
-inline double Potential(double data, std::vector< std::vector<double> > params,
+inline double Potential(double data, const double (*params)[8],
```

Comment:
could you please comment in the conversation why you decided to change the datatype of the input argument

Reply:
I had to change the type here, because I decided to store the constant values in the CalculatePseudoEnergy function in constant stack-allocated arrays over storing them in a `std::vector` for better memory efficiency. Since these stack-allocated arrays aren't of the datatype `std::vector<std::vector<double>>` anymore, I had to adjust the parameter type in the Potential function.